src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -8635,8 +8637,8 @@ namespace AirflowNetwork {
         }
 
         // Calculate sensible and latent loads in each zone from multizone airflows
-        if (SimulateAirflowNetwork == AirflowNetworkControlMultizone || SimulateAirflowNetwork == AirflowNetworkControlMultiADS ||
-            (SimulateAirflowNetwork == AirflowNetworkControlSimpleADS && AirflowNetworkFanActivated)) {
+        if (multizone_always_simulated || 
+            (simulation_control.type == ControlType::MultizoneWithDistributionOnlyDuringFanOperation && AirflowNetworkFanActivated)) {
```

Comment:
Some of the logic doesn't look too much better, but it is at least a little shorter.